js/apps/events-page/constants.js

```diff
@@ -2,8 +2,8 @@
 	'use strict';
 
 	var constants = {
-		baseUrl: 'https://testservices.bcpl.info',
-		// baseUrl: 'http://ba224964:3100',
+		// baseUrl: 'https://testservices.bcpl.info',
+		baseUrl: 'http://oit226471:1919',
```

Comment:
Please move the comment back so you're committing a file that looks at testservices.

Reply:
done